### PR TITLE
Update configuration.rst

### PR DESCRIPTION
**Pull request by fstr**
- Removed "manager" node. Does not exist anymore in \Sylius\Bundle\ResourceBundle\DependencyInjection\Configuration.php
- Fixed typo in driver name

### CODE REVIEW

File: bundles/SyliusResourceBundle/configuration.rst [hunk 1]
```diff
@@ -18,7 +18,6 @@ In your ``app/config.yml`` (or in an imported configuration file), you need to d
         resources:
             my_app.entity_key:
                 driver: doctrine/orm
-                manager: default
```
Comment (aRn0D) [inline bundles/SyliusResourceBundle/configuration.rst]: This is object_manager ;)

Comment (fstr) [inline bundles/SyliusResourceBundle/configuration.rst]: Actually I tried renaming it to object_manager with no success. 

[Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]             Unrecognized option "object_manager" under "sylius_resource.resources.my_app.entity_key"

Comment (aRn0D) [inline bundles/SyliusResourceBundle/configuration.rst]: You're right, this is a bug. Can you fix it ?

Comment (fstr) [inline bundles/SyliusResourceBundle/configuration.rst]: I'd just add a scalarNode called "manager" as a required node to \Sylius\Bundle\ResourceBundle\DependencyInjection\Configuration.php

If that's it, I will provide a fix.

**Comment by fstr** [comment]
PR can be dismissed. Documentation is up to date with dev-master.

I followed the manual:
$ composer require "sylius/resource-bundle"

Composer always installs the latest *stable* if you do not explicitly specify otherwise. So I ended up with 0.12.x instead of 0.13.x (dev-master) to which the manual refers. My bad.

See: https://groups.google.com/forum/#!topic/composer-dev/_g3ASeIFlrc/discussion

**Pull request closed without merging**